### PR TITLE
Refactor apiUmbrellaWeb usage

### LINKED ISSUE #1485: Refactor APIUmbrellaWeb object usage
Refactor createApiUmbrellaWeb function and apiUmbrellaSettingsValid functions.

createApiUmbrellaWeb returns API wrapper object. To be called when needed. Switching away from global object usage.

### PR DESCRIPTION
**Pull request by jykae**
Closes #1485 

## Proposed changes
- refactor createApiUmbrellaWeb to return wrapper object for Umbrella Admin API
- refactor apiUmbrellaSettingsValid function based on new field names in Proxies

Please review @apinf/developers

### CODE REVIEW

File: proxies/server/methods.js [hunk 1]
```diff
@@ -23,34 +24,25 @@ Meteor.methods({
     }
   },
   'createApiUmbrellaWeb': function () {
-    const settings = Settings.findOne();
+    // Assume one proxy only
```
Comment (brylie) [inline proxies/server/methods.js]: Add a TODO here mentioning that we need to refactor this method for multi-proxy support. That way we will remember what files to edit.

**Comment by brylie** [comment]
@jykae this looks good. Please add the TODO comment, and we can merge the PR.

**Comment by jykae** [comment]
@brylie TODO added

**Comment by brylie** [comment]
I added a commit to clarify the comment. I.e. a TODO should be a verb, something that we have *to do*.

**Comment by jykae** [comment]
Ok, thanks. Anything else or could we merge this so I could use it in apiKey PR ?

**Pull request merged**